search/src/main/kotlin/com/ritense/search/service/SearchFieldV2Service.kt

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2015-2022 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.search.service
+
+import com.ritense.search.domain.SearchFieldV2
+import com.ritense.search.repository.SearchFieldV2Repository
+import org.springframework.http.HttpStatus
+import org.springframework.web.server.ResponseStatusException
+
+class SearchFieldV2Service(
+    private val searchFieldV2Repository: SearchFieldV2Repository
+) {
+
+    fun create(searchFieldV2: SearchFieldV2) = searchFieldV2Repository.save(searchFieldV2)
```

Comment:
It should not be possible to create multiple search-fields with the same ownerId and key